### DISCUSSION TITLE
Please allow re-assigning Ctrl-Y

### DISCUSSION THREAD

**Discussion by JuriTuuti** [brackets-cont/brackets#309] [Feature Request]
For historical reasons, and finger habit, in every program I use, I have always assigned Ctrl-Y to _Delete Line_. Brackets is the only program I use that does not let me do that.

It is a finger habit that is hard to break, and find myself repeatedly unintentionally undeleting things.

**Comment by charlypa** [comment]
@JuriTuuti,

We've updated our editor to Phoenix Code, launched just yesterday. It now supports native Windows Mac and Linux, moving beyond the older Brackets x86 binaries. 

You can download  Phoenix Code at https://phcode.io/.

In Phoenix Code, editing keyboard shortcuts is straightforward—just click the keyboard icon in the menu. However, please note that Ctrl+Y might not be customizable due to its predefined OS function for Redo, but other keys can be reassigned as needed.
